js/coinspot.js

```diff
@@ -265,9 +267,6 @@ module.exports = class coinspot extends Exchange {
     }
 
     sign (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
```

Comment:
@kroitor although there's only one public method, we don't need credentials to access it, so I've removed this assertion.
If there's anything I'm not seeing, lmk, I'll add it back
![image](https://user-images.githubusercontent.com/43336371/147661904-6768fc0b-efb3-4ba1-b2a9-bfe81284f2a0.png)